.github/workflows/config/node-release.yaml

```diff
@@ -1,11 +1,11 @@
 release:
   branching:
     execution:
-      time: "20:00:00"
+      time: "17:00:00"
     schedule:
-      - on: "2024-07-26"
-        name: release/0.53
+      - on: "2024-08-30"
+        name: release/0.54
         initial-tag:
           create: false
-          name: v0.53.0-alpha.0
+          name: v0.54.0-alpha.0
```

Comment:
Since the initial-tag create value is `false`, the `v0.54.0-alpha.0` value is not used. Is this what you intended?

Reply:
You're right, I read this incorrectly. For some reason I was thinking it would create the branch from the given tag, but that's not what this does.